automation/submission-perf-test.sh

```diff
@@ -9,13 +9,50 @@ WORKING_DIR=${3:-$PWD}
 
 # Check if $ENV is not empty
 if [[ ! ${ENV} ]]; then
-    echo "ENV is empty. Run again \`sh submission-perf-test.sh <alpha or staging>\`"
+    echo
+    "ENV is empty. Run again sh submission-perf-test.sh <alpha or staging"
     exit 1
-else echo "Starting Perf test in $ENV"; fi
-
+else
+    echo "Starting Perf test in {$ENV}"
+fi
 
 JSON_CREDS=`docker run --rm -e VAULT_TOKEN=$VAULT_TOKEN -e VAULT_ADDR=https://clotho.broadinstitute.org:8200 broadinstitute/dsde-toolbox vault read -format=json secret/dsde/firecloud/dev/common/firecloud-account.pem | jq '.data'`
 
+users=(
+     harry.potter@test.firecloud.org
+     ron.weasley@test.firecloud.org
+     mcgonagall.curator@test.firecloud.org
+     draco.malfoy@test.firecloud.org
+     hermione.owner@test.firecloud.org
+     dumbledore.admin@test.firecloud.org
+   )
+
+
+checkToken () {
+    user=$1
+
+    # Verify that user does not need to refresh their token
+    if
+        curl -f -v --silent -X GET --header "Accept: application/json" --header "Authorization: Bearer $ACCESS_TOKEN" "https://firecloud-orchestration.dsde-$ENV.broadinstitute.org/api/refresh-token-status"  2>&1 | grep '"requiresRefresh": true'
+    then
+        echo "This user needs its refresh token refreshed"
+        exit 1
+    fi
```

Comment:
As mentioned previously, I think it would be useful for the code to check ALL users' tokens in `checkToken`. If any need refreshing, then print out the names of each user and then `exit 1`.

Reply:
I agree with that, will change